README.md

```diff
@@ -67,6 +67,11 @@ Upserts all data types without specifying a schema
 
 ### three_operations
 Shows how to use upsert, update and delete operations
+
+### priority_first_sync_for_high_volume_initial_syncs
+Priority-first sync, pfs for short, is very helpful for high volume historical syncs. It is a sync strategy
```

Comment:
I think we should include a link to our docs on the approach:
https://fivetran.com/docs/using-fivetran/features#priorityfirstsync

Reply:
Varun suggested not to. Replied in slack.

Reply:
Ok for now then.